integration/deployment/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh -e
+DANMKUBECONFIG=/etc/kubernetes/danm.kubeconfig
+cp /opt/bin/* /target/cni-bin/
```

Comment:
the script needs to start with checking whether danm is already installed on the cluster, and should exit immediately if it is.
we should not overwrite existing installations

checking the presence of danm CNI, and 00-danm.conf is suitable